components/CommonHeader.tsx

```diff
@@ -106,62 +189,26 @@ const CommonHeader = () => {
               padding: '1.25rem 2.5rem !important',
             }}
           >
-            <LogoBox/>
+            <LogoBox />
             <nav>
               <ul>
```

Comment:
I think there should be a little gap between the links. 

I played a bit with dev tools. You could replace the ul with something like `<List sx={{display: 'flex', gap: '0.5rem'}}>`

![image](https://github.com/user-attachments/assets/876103d9-cb63-4360-a8e9-7a24f52d2876)

Reply:
thanks again